app/src/main/java/com/example/chrono/main/timer/CircuitCreate.kt

```diff
@@ -94,25 +96,25 @@ class CircuitCreate : BaseActivity() {
     }
 
     private fun addSet() {
-        if (bind!!.setNum.text.toString() == "") {
-            bind!!.setNum.setText("1")
+        val currentText = bind!!.setNum.text.toString()
+        if (currentText == "") {
+            setSetNum(1)
         } else {
-            bind!!.setNum.setText((bind!!.setNum.text.toString().toInt() + 1).toString())
```

Comment:
One of the advantages of using data binding is the ability to just access views with bind. instead of creating a new variable for it etc. That's why I'm always accessing views through bind and not creating a separate variable. Saves lines of code and is still easily readable to anyone who knows about Android/binding

Reply:
Don't bother changing these now, but moving forward no need to create variables for views, that sort of defeats the purpose of bind.

Reply:
kk